shell/shared/grain.js

```diff
@@ -1358,6 +1432,19 @@ if (Meteor.isClient) {
             return "remove";
           }
         });
+      } else if (event.data.inlinePowerbox) {
+        var inlinePowerbox = event.data.inlinePowerbox;
+        check(inlinePowerbox, Object);
+        var rpcId = inlinePowerbox.rpcId;
```

Comment:
I'm inclined to say that we should either define `rpcId` to be `String` or we should simply let it be `Any`.  `rpcId` exists solely so that the app can give a unique identifier to the shell with a request so it can tell which request a `postMessage()`'d reply refers to, if there are multiple requests away at once.

Given that, rpcId might as well be `Any` - it's an opaque blob, the equivalent of a void* userdata in a struct.